### DISCUSSION TITLE
Display Value of State in Paragraph Component

### DISCUSSION THREAD

**Discussion by chadgriff** [Budibase/budibase#6297] [Help] [answered]
I am having trouble displaying a value of a column in a SQL Server table relationship column.  What is displayed is the full object and I cannot seem to get any handlebars to pull the desired substring from the object. 

What is being displayed on screen is as follows:
`[{"primaryDisplay":"Sales Rep","_id":"%5B7000014%5D"}]`

What I would like to have displayed is:
`Sales Rep`

The paragraph component is set as follows:
`Company Name: {{State.applicant_salesRep}}`

The value of the state is set as follows:
![image](https://user-images.githubusercontent.com/15015476/173391471-4bf08707-b87c-44c3-b97f-e4b23e1590e7.png)

Any help would be appreciated...thanks!

**Comment by melohagan** [comment, marked as answer]
Hi @chadgriff 

The handlebars expression would be: `{{ Dealer Table.dealers.fk_salesReps.0.primaryDisplay }}`

Relationships are always stored in an array, even if it is a one-one.

**Reply by chadgriff** [reply]
Thanks @melohagan ... thats what I was missing!
